scout/server/blueprints/cases/templates/cases/cases.html

```diff
@@ -218,7 +218,9 @@ <h5 class="modal-title" id="sangerModalLabel">Sanger validations to evaluate:</h
     </td>
     <td>
       {{ case.analysis_date.date() }}
-      {% if case.is_rerun %}
+      {% if case.rerun_requested %}
+          <span class="badge badge-secondary pull-right">rerun pending</span>
+      {% elif case.is_rerun %}
         <span class="badge badge-secondary pull-right">rerun</span>
```

Comment:
Same here, it will be fixed by changing the class of the td as well

Reply:
Looks like float-* is going to remain supported, so for simplicity I switched to that instead!